content/SCALE/SCALETutorials/Shares/MixedModeShares.md

```diff
@@ -0,0 +1,238 @@
+---
+title: "Multiprotocol Shares"
+description: "Provides instructions on setting up SMB and NFSv4 mixed-mode shares."
+weight: 40
+tags:
+- scaleshares
+- scalesmb
+- scalenfs
+---
+
+{{< toc >}}
+
+## About Multiprotocol Shares
+
+A multiprotocol or mixed-mode NFS and SMB share supports both NFS and SMB protocols for sharing data.
+Multiprotocol shares allow clients to use either protocol to access the same data.
+This can be useful in environments with a mix of Windows systems and Unix-like systems, especially if some clients lack an SMB client.
+
+{{< hint type=tip >}}
+Carefully consider your environment and access requirements before configuring a multiprotocol share.
+For many applications, a single protocol SMB share provides better user experience and ease of administration. Linux clients can access SMB shares using [`mount.cifs`](https://linux.die.net/man/8/mount.cifs).
+{{< /hint >}}
+
+It is important to properly configure permissions and access controls to ensure security and data integrity when using mixed-mode sharing.
+To maximize security on the NFS side of the multiprotocol share, we recommend using NFSv4 and [Active Directory](#joining-active-directory) for Kerberos authentication.
+It is also important that NFS clients preserve extended attributes when copying files, or SMB metadata could be discarded in the copy.
+
+## Adding a Multiprotocol Share
+
+Adding a multiprotocol SMB and NFS share to your system involves several steps:
+
+1. [Start and configure](#starting-and-configuring-services) the SMB and NFS services.
+
+2. Join the TrueNAS server to an existing [Active Directory](#joining-active-directory) domain.
+   Configure a container, Kerberos admin, and user accounts in AD.
+
+3. [Set up a dataset](#creating-a-multiprotocol-share-dataset) for the new share with **Share Type** set to **Multiprotocol**.
+
+4. [Create the SMB share](#creating-the-smb-share) with **Purpose** set to **Multi-protocol (NFSv4/SMB) shares**.
+
+5. [Create the NFS share](#creating-the-nfs-share) with **Security** set to **KRB5**.
```

Comment:
Same comment as the SMB share step. Can you link to the "AddingNFSShares.md" article?